aiida_quantumespresso/workflows/pw/band_structure.py

```diff
@@ -20,8 +20,8 @@ def define(cls, spec):
         super(PwBandStructureWorkChain, cls).define(spec)
         spec.input('code', valid_type=orm.Code)
         spec.input('structure', valid_type=orm.StructureData)
-        spec.input('protocol', valid_type=orm.Dict)
-        spec.input('scf_options', valid_type=orm.Dict)
+        spec.input('protocol', valid_type=orm.Dict, required=False, default=orm.Dict(dict={'name':'theos-ht-1.0'}))
```

Comment:
specifying a default will automatically set `required=False` so explicitly defining it here is superfluous

Reply:
I prefer it to be explicit, but I don't really mind. I'll remove it.

And we tested it on the tutorial VMs: it seems to work, although we couldn't complete the calculations for other reasons.

Reply:
Explicit is good, but in this case it is really superfluous. If you provide a default, it is automatically not required, since if not specified the default will be used, by definition